src/api/java/cvc5/DriverOptions.java

```diff
@@ -0,0 +1,53 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Gereon Kremer, Mudathir Mohamed
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * The cvc5 java API.
+ */
+
+package cvc5;
+
+/**
+ * Provides access to options that can not be communicated via the regular
+ * getOption() or getOptionInfo() methods. This class does not store the options
+ * itself, but only acts as a wrapper to the solver object. It can thus no
+ * longer be used after the solver object has been destroyed.
+ */
+public class DriverOptions extends AbstractPointer
```

Comment:
I thought the same, thanks for the confirmation. However, it is possible to redirect the streams from C++ to Java streams and vice versa, but it is a hassle. None of the streams is implemented in the Java API now. So this may be a place holder for when streams are supported. Do you still want it to be removed?

Reply:
I had a short discussion with @4tXJ7f about that. We figured that it may be possible to wrap the streams, but that it would probably not that easy. In particular, note that the `DriverOptions` allows to **set** streams as well (they are non-const references), which is intentional.
If you have a clear idea on how to achieve this, it would be awesome to have it. But we are fine with just not supporting this in the java API, and then not having the DriverOptions object.

Reply:
It seems there is a way to support streams with buffers as described in this article, 
https://codeforfun.wordpress.com/2010/04/22/sharing-output-streams-through-a-jni-interface/
but we can support this in the future.

Reply:
Okay, it seems doable, but I think there are enough technical details to warrant a separate PR. Also, we should figure out whether we actually want to support this... @4tXJ7f